core/src/main/java/org/kiwix/kiwixmobile/core/utils/KiwixDialog.kt

```diff
@@ -178,7 +178,7 @@ sealed class KiwixDialog(
     )
 
     object OpenInNewTab : YesNoDialog(
-      R.string.open_in_new_tab, R.string.open_in_new_tab
+      R.string.open_in_new_tab, R.string.confirm_open_in_new_tab
```

Comment:
set the title to null and use `open_in_new_tab` for the message, this dialog is too redundant

Reply:
done